Auto-generate ng_crf_penalty_st.csv and update financials.py (#113)
ReEDS-Model/ReEDS: docs/sources_documentation.md

  - [hydrofrac_policy.csv](/inputs/state_policies/hydrofrac_policy.csv)
---

  - [ng_st_forced.csv](/inputs/state_policies/ng_st_forced.csv)
    - **File Type:** Inputs

>     - **Description:** The certain year for specific states that NG techs must be retired by a certain year

Reviewer: ```suggestion
    - **Description:** The year for specific states in which natural gas techs must be retired
```
